tests/chaos/oadp/conftest.py

```diff
@@ -64,3 +74,14 @@ def rebooted_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress, wo
     LOGGER.info(f"Waiting for node {vm_node.name} to come back online")
     wait_for_node_status(node=vm_node, status=True, wait_timeout=TIMEOUT_10MIN)
     return
+
+
+@pytest.fixture()
+def drain_vm_source_node(rhel_vm_with_dv_running, oadp_backup_in_progress):
+    vm_node = rhel_vm_with_dv_running.vmi.node
+    # Ensure that backup has been in the InProgress status and then drain the node
+    _ = oadp_backup_in_progress
+    time.sleep(TIMEOUT_5SEC)
+    with node_mgmt_console(node=vm_node, node_mgmt="drain", timeout=TIMEOUT_10MIN):
+        wait_for_node_schedulable_status(node=vm_node, status=False, timeout=TIMEOUT_5MIN)
```

Comment:
are you sure you need to increase the timeout? we have "drain" in many places and it always was fine to get the node in "False" state with the default timeout